src/components/client/campaigns/CampaignNewsSection.tsx

```diff
@@ -0,0 +1,314 @@
+import { Typography, Grid, Button, Link } from '@mui/material'
+import { CampaignResponse } from 'gql/campaigns'
+import {
+  Timeline,
+  TimelineItem,
+  TimelineSeparator,
+  TimelineConnector,
+  TimelineContent,
+  TimelineDot,
+  TimelineOppositeContent,
+} from '@mui/lab'
+import AvTimerIcon from '@mui/icons-material/AvTimer'
+import SupervisedUserCircleOutlinedIcon from '@mui/icons-material/SupervisedUserCircleOutlined'
+
+import { styled } from '@mui/material/styles'
+import { dateToTime, formatDateString } from 'common/util/date'
+import { routes } from 'common/routes'
+import { OutlinedButton } from '../index/IndexPage.styled'
+
+import { useTranslation } from 'next-i18next'
+
+import Image from 'next/image'
+import useMobile from 'common/hooks/useMobile'
+import { GetArticleDocuments, GetArticleGalleryPhotos } from 'common/util/newsFilesUrls'
+import { useShowMoreContent } from '../campaign-news/hooks/useShowMoreContent'
+import { sanitizeHTML } from 'common/util/htmlSanitizer'
+
+const PREFIX = 'NewsTimeline'
+
+const classes = {
+  timelineItem: `${PREFIX}-timelineItem`,
+  connector: `${PREFIX}-connector`,
+  dot: `${PREFIX}-dot`,
+  timelineOppositeContent: `${PREFIX}-timelineOppositeContent`,
+  articlePublishedDate: `${PREFIX}-articlePublishedDate`,
+  articleAuthor: `${PREFIX}-articleAuthor`,
+  articleContent: `${PREFIX}-articleContent`,
+  timelineContent: `${PREFIX}-timelineContent`,
+  articleHeader: `${PREFIX}-articleHeader`,
+  articleDescription: `${PREFIX}-articleDescription`,
+  articleAttachmentContainer: `${PREFIX}-articleAttachment`,
+  readMoreButton: `${PREFIX}-readMoreButton`,
+  readAllButton: `${PREFIX}-readAllButton`,
+  dateAndAuthorContainer: `${PREFIX}-dateAndAuthorContainer`,
+}
+
+const StyledTimeline = styled(Timeline)(({ theme }) => ({
+  padding: 0,
+  [`& .MuiTimelineItem-root:before`]: {
+    display: 'none',
+  },
+
+  [`& .${classes.connector}`]: {
+    backgroundColor: theme.palette.primary.light,
+    minHeight: theme.spacing(20),
+  },
+
+  [`& .${classes.dot}`]: {
+    backgroundColor: theme.palette.primary.light,
+    margin: 0,
+  },
+
+  [`& .${classes.dateAndAuthorContainer}`]: {
+    marginBottom: theme.spacing(2),
+  },
+
+  [`& .${classes.timelineOppositeContent}`]: {
+    paddingTop: theme.spacing(0),
+    paddingLeft: theme.spacing(0),
+    maxWidth: theme.spacing(18),
+    marginTop: theme.spacing(-0.5),
```

Comment:
AFAIK negative margins should be avoided

Reply:
Without this negative margin there is a slight offset between the beginning of the content and the beginning of the timeline component(the dot). Adding any sort of margin to the `Timeline` component breaks it, thus the need to adjust the content, through negative margin. 

I could remove the negative margin if that offset doesn't bother anyone, but so far it doesn't seem to cause any conflict with other DOM elements, in either desktop or mobile viewports.